README.md

```diff
@@ -81,13 +81,15 @@ _Helpful to write SNMP applications._
   - There is also a [community fork](https://github.com/pysnmp/pysnmp) under active development, but it is not clear if that project is commited to maintaining all the associated repos, documentation, and websites.
 - [pysmi](https://github.com/etingof/pysmi) - PySMI is a pure-Python implementation of SNMP SMI MIB parser.
   - Abandoned for the same reason as the above pysnmp project. See [lextudio/pysmi](https://github.com/lextudio/pysmi).
+- [gufo_snmp](https://github.com/gufolabs/gufo_snmp) - The accelerated Python SNMP client library supporting both async and synchronous mode. It consists of a clean Python API for high-efficient BER parser and socket IO, implemented in the Rust language with PyO3 wrapper. Seems to be a bit early in the project's lifecycle, but it is easy to use and ***extremely*** fast, especially when querying many devices.
```

Comment:
awesome-lint complains about "***extremely***".

   README.md:84:352
  ✖  84:352  Emphasis should use _ as a marker  remark-lint:emphasis-marker

Let's try to follow awesome-lint's advice.

Reply:
Changed per the linter's suggestion.

Reply:
Also fixed a few other places in the document that the linter was complaining about.